### ISSUE TITLE
Setting to change ordering of items and amount of displayed items

### ISSUE THREAD

**Issue by nilithu** [thatsmybis/thatsmybis#73]
When distributing loot based on priorities of items (wishlist item 2 > 5) it is currently challenging to quickly assess who has the highest priority due to the alphabetical ordering of characters in certain views (loot & prio view)

Also the limitation of items shown per default (currently only 4 items are shown in various views per and require a click on "show X more items") to display the whole list. This is no problem for a few people but difficult for top items where 15+ people are displayed)

**Describe the solution you'd like**
An opinion on the guild level that allows to define the ordering logic on the prio and loot views. 

Per default, nothing should change. If activated order the characters based on the items index in the characters wishlist. 

Another option would be required to toggle the "view X more items" or to display everything per default.

**Comment by Lemmings19** [comment]
I've added a button to toggle to show all loot. I won't be adding a default guild feature for this for the time being, but this should still save you a lot of headache.
![image](https://user-images.githubusercontent.com/3407841/95168150-baae2b80-0765-11eb-9445-497a346ad6b9.png)

I've changed wishlists to now be sorted by the order that people prioritized items. This should take effect across the whole site. Hopefully it didn't break anything or make anything worse, but if it did just let me know.
